src/index.js

```diff
@@ -0,0 +1,182 @@
+        image2.src="images/winter/mountainCabin.jpg";
+        image3.src="images/winter/snowman.jpg";
+        image4.src="images/winter/snowTrees.jpg";
+    } else if(60 <= state.temp && state.temp <= 69){
+        image1.src="images/autumn/autumnTreesRoad.jpg";
+        image2.src="images/autumn/pumpkinBroom.jpg";
+        image3.src="images/autumn/mushrooms.jpg";
+        image4.src="images/autumn/autumnTreesStairs.jpg";
+    } else if(70 <= state.temp && state.temp <= 79){
+        image1.src="images/spring/beeOnSunflower.jpg";
+        image2.src="images/spring/flowers.jpg";
+        image3.src="images/spring/rain.jpg";
+        image4.src="images/spring/tulipAirBalloon.jpg";
+    } else if (state.temp >= 80){
+        image1.src="images/summer/sunglassesOnBeach.jpg";
+        image2.src="images/summer/boatOnOcean.jpg";
+        image3.src="images/summer/palmTrees.jpg";
+        image4.src="images/summer/sandalsOnBeach.jpg";
+    }
+};
+
+const input = document.querySelector("input");
+const city = document.getElementById("cityName");
+input.addEventListener("input", () => {
+    state.changeCity = document.getElementById("city").value;
+    document.getElementById("cityName").innerHTML = state.changeCity;
+});
+
+const reset = document.getElementById("resetBtn");
+const resetValue = input.getAttribute("placeholder");
+reset.addEventListener("click", () => {
+    input.value = resetValue;
+    city.textContent = resetValue;
+})
+
+const getRealTemp = () => {
+    state.changeCity = document.getElementById("city").value;
+    getLatLon();
+}
+
+const changeSky = () => {
+    state.skyCondition = document.getElementById("tempContainer");
+    if (document.getElementById("skyOptions").value === "sunny") {
+        state.skyCondition.style.background = "rgba(255, 240, 0, 0.8)";
+
+    } else if (document.getElementById("skyOptions").value === "rainy") {
+        state.skyCondition.style.background = "rgba(137, 196, 244, 0.8)";
+
+    } else if (document.getElementById("skyOptions").value === "cloudy") {
+        state.skyCondition.style.background = "rgba(0, 0, 0, 0.5)";
+
+    } else if (document.getElementById("skyOptions").value === "snowy") {
+        state.skyCondition.style.background = "rgba(255, 255, 255, 0.8)";
+
+    }
+};
+
+const getLatLon = () => {
+    axios.get("http://127.0.01:5000/location", {
+            params: {
+                q: state.changeCity,
+            },
+        })
+        .then((response) => {
+            console.log(response);
+            state.lat = response.data[0].lat;
+            state.lon = response.data[0].lon;
+            console.log("Success finding latitude and longitude!");
+            getWeather(state.lat, state.lon);
+
+        })
+        .catch((error) => {
+            console.log("Error finding latitude and longitude!");
+            console.log(error.message);
+        });
+};
+
+const kelvinToFahrenheit = (kelvin) => {
+    return Math.floor(1.8 * (kelvin - 273) + 32);
+};
```

Comment:
Nice, I like this pulled into a helper method even though it's not a long method